RDoc-2562 Cloud ->Microsoft Azure Marketplace - prepare a documentation about the integration between RavenDB Cloud and Azure Marketplace (#1822)
ravendb/docs: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-microsoft-azure-marketplace.markdown

﻿# Cloud: Microsoft Azure Marketplace
---

{NOTE: }

Access to *RavenDB Cloud* services is available through the *Microsoft Azure Marketplace*.

* In this page:
    * [Overview](../cloud/cloud-microsoft-azure-marketplace#overview)
    * [Registration process](../cloud/cloud-microsoft-azure-marketplace#registration-process)
    * [Subscription cancellation process](../cloud/cloud-microsoft-azure-marketplace#subscription-cancellation-process)

{NOTE/}

---

{PANEL: Overview}

If you want to pay for *RavenDB Cloud* services using a credit card associated with your *Microsoft Azure* account, without re-entering the card in *RavenDB Cloud*, you can subscribe to [RavenDB Cloud offer](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/hibernatingrhinos.ravendb-cloud-payg?tab=Overview) on the *Microsoft Azure Marketplace*.

> The *Microsoft Azure Cloud Accounts* created by this integration have a few differences compared to the regular *Cloud Accounts*:

Reviewer: ```suggestion
*Microsoft Azure Cloud Account* created this way has a few differences in comparison to the regular *Cloud Account*:
```

> - Monthly invoices are issued by *Microsoft Azure*, not by our standard billing system.

Reviewer: ```suggestion
- Monthly invoices are issued by *Microsoft Azure*, not by RavenDB Cloud
```

> - *Microsoft Azure Cloud Accounts* are permitted to provision only *Azure* products. Regular Cloud Accounts can also provision *AWS* and *GCP* products.

Reviewer: ```suggestion
- *Microsoft Azure Cloud Account* offering is limited to *Azure* products.
```

!["Figure 1 - RavenDB Cloud offer in Microsoft Azure Marketplace"](images\microsoft-azure-marketplace-cloud-ravendb-offer.png "Figure 1 - RavenDB Cloud offer in Microsoft Azure Marketplace")

---

In the main page of the offer, click **Subscribe**.

!["Figure 2 - Main page of the RavenDB Cloud offer in Microsoft Azure Marketplace"](images\microsoft-azure-marketplace-cloud-ravendb-offer-overview.png "Figure 2 - Main page of the RavenDB Cloud offer in Microsoft Azure Marketplace")

---

Provide the necessary information required to subscribe to the offer:

!["Figure 3 - RavenDB Cloud subscription configuration panel"](images\microsoft-azure-marketplace-cloud-ravendb-offer-configration.png "Figure 3 - RavenDB Cloud subscription configuration panel")

1.  **Subscription**  
    Select the *Microsoft Azure subscription* from which your payment information will be used for this offer.

2.  **Resource group**  
    Select the *Microsoft Azure resource group* where the subscription will be located.

3.  **Name**  
    Provide a name for the subscription. 

> 4.  **Recurring billing**  

Reviewer: Let's put an asterisk on it for attention? Thoughts?
```suggestion
4.  **Recurring billing**  *
```

Author: Ok, added.

    When recurring billing is on, your subscription will renew at the end of the billing term.

{DANGER: }
If **recurring billing** option is **off**, your subscription **won't be renewed at the end of the billing term**.

> This means your subscription, account and related Cloud products **will be permanently lost** at the end of the billing term.

Reviewer: ```suggestion
This means your subscription, account along with its Cloud products and data they held, **will be permanently lost** at the end of the billing term.
```

{INFO: }
Providing *Primary phone number* **is not required**.
{INFO/}

{WARNING: }
*Registration process* requires *Microsoft Azure* account with **configured email**.
{WARNING/}

After reviewing the information, please click **Subscribe**.

---

Once you have subscribed, the next step is to configure the account in the *RavenDB Cloud*.

!["Figure 5 - RavenDB Cloud subscription panel - redirect to the RavenDB Cloud page"](images\microsoft-azure-marketplace-cloud-ravendb-offer-almost-done-configuration.png "Figure 5 - RavenDB Cloud subscription panel - redirect to the RavenDB Cloud page")

---

Provide the necessary information required to end the registration process:

!["Figure 6 - RavenDB Cloud Azure Marketplace Landing Page"](images\cloud-ravendb-azure-marketplace-landing-page.png "Figure 6 - RavenDB Cloud Azure Marketplace Landing Page")

1.  **Domain Name**  
    Provide a Domain Name for the account.

2.  **Account Type**  
    Provide an Account Type for the account.

3.  **T&C and privacy policy**  

>     Accept Terms & Conditions and our privacy policy.

Reviewer: ```suggestion
    Accept Terms & Conditions and the privacy policy.
```

After providing the information, please click **Sign up**.

---

> Once you have done this, your account is ready to use. To sign in to the account, click **Continue to sign in**.

Reviewer: ```suggestion
Your account is now ready to use. To sign in to the account, click **Continue to sign in**.
```
